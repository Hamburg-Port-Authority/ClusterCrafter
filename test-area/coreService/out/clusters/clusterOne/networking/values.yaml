ingress-nginx:
  controller:
    image:
      repository: registry.k8s.io/ingress-nginx/controller
    admissionWebhooks:
      patch:
        image:
          repository: registry.k8s.io/ingress-nginx/kube-webhook-certgen
  defaultBackend:
    image:
      repository: registry.k8s.io/defaultbackend-amd64