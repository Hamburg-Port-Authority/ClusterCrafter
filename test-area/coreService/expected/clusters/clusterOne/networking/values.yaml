ingress-nginx:
  controller:
    image:
      repository: registry.k8s.io/ingress-nginx/controllerasdasdasd
    admissionWebhooks:
      patch:
        image:
          repository: registry.k8s.io/ingress-nginx/kube-webhook-certgen
  defaultBackend:
    image:
      repository: registry.k8s.io/defaultbackend-amd64